openpype/settings/defaults/project_settings/maya.json

```
@@ -1047,6 +1047,10 @@
                 125,
                 255
             ]
+        },
+        "reference_loader": {
+            "namespace": "{asset_name}_{subset}_##",
```

To match the original behavior I believe this should still end with a `_` after the digits.

```suggestion
            "namespace": "{asset_name}_{subset}_##_",
```